src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE hibernate-configuration PUBLIC
+        "-//Hibernate/Hibernate Configuration DTD 3.0//EN"
+        "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+
+<hibernate-configuration>
+    <session-factory>
+        <property name="dialect">org.hibernate.dialect.MariaDBDialect</property>
```

Comment:
why do you use `MariaDBDialect`?

Reply:
i like to use it

Reply:
why?
is it better than MySQLDialect? 
please, provide examples when MariaDBDialect is better